src/__tests__/transformer.test.ts

```diff
@@ -62,6 +62,46 @@ describe("Responsive Variants", () => {
     expect(result).toBe(expectedContent(sourceCode, transformedContent));
   });
 
+  test("should return a transformed content (string) for a class with ')'", () => {
+    const sourceCode = `
+    import { tv } from "tailwind-variants";
+
+    
+    export const dialog = tv(
+      {
+        variants: {
+          color: {
+            primary: "w-[calc(100%-1rem)]"
+          }
+        }
+      },
+      {
+        responsiveVariants: true
+      },
+      { responsiveVariants: ['sm'] },
```

Comment:
Hey @PatrykWalach thank you for your contribution 🙌🏻 could you please remove this invalid param? 👀